scripts/admin/launch-meteor

```diff
@@ -82,15 +82,36 @@ if [ ! -x "$METEOR_WAREHOUSE_DIR/meteor" ]; then
   fi
   echo "Installing a Meteor distribution in your home directory." 1>&2
 
-  # Only show progress bar animations if we have a tty
-  # (Prevents tons of console junk when installing within a pipe)
-  if [[ -t 1 ]]; then
-    curl --progress-bar --fail "$TARBALL_URL" | tar -xzf - -C "$INSTALL_TMPDIR"
-  else
-    curl -s --fail "$TARBALL_URL" | tar -xzf - -C "$INSTALL_TMPDIR"
-  fi
+  TARBALL_FILE="$(dirname "$METEOR_WAREHOUSE_DIR")/.meteor-tarball-tmp"
+  rm -rf "$TARBALL_FILE"
+
+  echo "Downloading Meteor distribution"
+  # keep trying to curl the file until it works (resuming where possible)
+  MAX_ATTEMPTS=99
+  set +e
+  CODE=1
+  ATTEMPTS=0
+  while [ $CODE -ne 0 ] && [ $ATTEMPTS -lt $MAX_ATTEMPTS ]
+  do
+    ATTEMPTS=$((ATTEMPTS + 1))
+
+    # Only show progress bar animations if we have a tty
+    # (Prevents tons of console junk when installing within a pipe)
+    if [[ -t 1 ]]; then
```

Comment:
Btw, on the `install.meteor.com` script, I just noticed that you completely removed the "IF TTY USE `progress-bar`" conditional.  You should be fine to continue using that so long as you only use single brackets instead of double-brackets:

    if [ -t 1 ]; then

Which I can confirm works in Debian properly when running the command without an interactive terminal (via SSH).  I didn't test with anything else (Docker, for example), but it should be the same as there is no STDIN descriptor.

(Also, I realize this is the wrong issue to discuss the `install.meteor.com` script on, however I don't have a better PR for it since it's not in the repo. :wink:)